src/main/java/org/jenkinsci/test/acceptance/Matchers.java

```diff
@@ -59,7 +58,7 @@ public void describeMismatchSafely(WebDriver item, Description mismatchDescripti
             @Override
             public boolean matchesSafely(WebDriver item) {
                 try {
-                    item.findElements(selector);
+                    item.findElement(selector);
```

Comment:
Indeed this matcher was flawed, and we use it in the new implementation of the check so we needed to fix it.

The problem in question is that `findElements` (plural) never throws a `NoSuchElementException`, just returns an empty list if no matches.